string recognized_word
float64 confidence
